mne/gui/tests/test_coreg_gui.py

```diff
@@ -196,10 +198,11 @@ def test_coreg_model_with_fsaverage():
 
     # test switching raw disables point omission but leaves EEG selector as is
     assert_equal(model.omitted_info['n_total'], 62)  # 1 HSP (dist), 61 EEG
-    with warnings.catch_warnings(record=True):
+    with warnings.catch_warnings(record=True):  # Traits comp to None
```

Comment:
FYI this existing line was a hint about the nature of the potential fix, I added a comment that hopefully makes it clearer